01/Evgeny Vasiukevich/src/components/HomesGuestsLoves/HomeItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { THomes } from '../../types/THomes';
+
+const HomeItem = ({key, home}: {key: string, home: THomes}) => {
```

Comment:
please, write type for props separately` type Props = { ...`

Reply:
Done